index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <style>
```

Comment:
Have you thought about using a styles.css sheet to separate the structure of the page and the style?